src/lib/components/Search/Search.svelte

```diff
@@ -0,0 +1,58 @@
+<script lang="ts">
+  import { createEventDispatcher } from "svelte";
+  // import searchIcon from "$icons/search--white.svg";
```

Comment:
assuming this is just dependent on #54 going in first?

Reply:
Yup, I'll fix this! Good catch.